src/Api/MailingList/Member.php

```diff
@@ -195,7 +198,7 @@ public function update(string $list, string $address, array $parameters = [])
         Assert::stringNotEmpty($address);
         Assert::isArray($parameters);
 
-        foreach ($parameters as $field => $value) {
+        foreach ($parameters as $field => &$value) {
```

Comment:
Why adding the `&` here? I can't see any need for this

Reply:
We are [encoding](https://github.com/mailgun/mailgun-php/blob/master/src/Api/MailingList/Member.php#L202) vars parameters but original value remains the same. It will cause exception #638 if `vars` will be passed like array

Reply:
We are using `&` [here](https://github.com/mailgun/mailgun-php/blob/master/src/Api/MailingList/Member.php#L147)

Reply:
I don't like to pass variables by reference (adding the `&`) unless it is strictly necessary, and I am not sure that this is the case here.

However, I am happy to keep it as far as it doesn't have any direct impact when managing the data in `$value`.